sections/04_study.md

```diff
@@ -389,7 +390,7 @@ align sequences and predict bulges, mismatches, and wobble base pairing without
 requiring the user to input secondary structure predictions or thermodynamic
 calculations.
 
-Further incremental advances in neural-network approaches for miRNA and target
+Further incremental advances in neural network approaches for miRNA and target
```

Comment:
I looked at the rest of the paragraph, and now I understand.  This one should be DL.